packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -71,10 +71,14 @@ enum GroupingType {
 const stringFormat = (value: string) => `'${value}'`;
 
 export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
-    public static readonly OPTIONS_CLONE_OPTS: CloneOptions = {
+    public static readonly OPTIONS_CLONE_OPTS_SLOW: CloneOptions = {
         shallow: new Set(['data', 'container']),
         assign: new Set(['context', 'theme']),
     };
```

Comment:
Some `agCharts.test.ts` test-cases are failing without this in the main options process path for `updateDelta()` cases - I want to tackle that distinctly.